datamade/la-metro-councilmatic
# Rotate old AWS keys

## Overview

LA Metro is still using old AWS credentials attached to evz's account in order to query the property image cache. Rotate these credentials for a new pair that is created for a separate IAM user and that only has read/write access to the two PIC buckets.

### Checklist


## Testing Instructions

 * Swap out your settings for the production settings:

```bash
blackbox_cat configs/settings_deployment.production.py.gpg > councilmatic/settings_deployment.py
```

* Is it safe to actually refresh the PIC cache? If so, run the management command and confirm that it works with the new credentials:

```
python manage.py refresh_pic
```

* If not, edit the `refresh_pic.py` management command in your virtualenv to comment out [the line that deletes keys from the cache](https://github.com/datamade/django-councilmatic/blob/8d64575cf93470d13db99db9f4d42afd3a67cc5c/councilmatic_core/management/commands/refresh_pic.py#L33), and then run the command

Connects https://github.com/datamade/ops/issues/579
